projects/plugins/jetpack/tests/e2e/bin/container-setup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
```

Comment:
```suggestion
set -eo pipefail
```
That makes it also exit if an earlier process in a pipe fails.